test(jans-auth-server): fix AuthorizationCodeFlowEmbeddedTest

### What is failing?
AuthorizationCodeFlowEmbeddedTest

### Where is it failing?
It is failing on jenkins

```
08:55:09,759 [ERROR] COMPILATION ERROR : 
08:55:09,759 [INFO] -------------------------------------------------------------
08:55:09,760 [ERROR] /home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/src/test/java/io/jans/as/server/ws/rs/AuthorizationCodeFlowEmbeddedTest.java:[603,59] cannot find symbol
  symbol:   method getCleanServiceInterval()
  location: variable appConfiguration of type io.jans.as.model.configuration.AppConfiguration
08:55:09,760 [ERROR] /home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/src/test/java/io/jans/as/server/ws/rs/AuthorizationCodeFlowEmbeddedTest.java:[607,25] cannot find symbol
  symbol:   method setCleanServiceInterval(int)
  location: variable appConfiguration of type io.jans.as.model.configuration.AppConfiguration
08:55:09,761 [ERROR] /home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/src/test/java/io/jans/as/server/ws/rs/AuthorizationCodeFlowEmbeddedTest.java:[659,29] cannot find symbol
  symbol:   method setCleanServiceInterval(int)
```


### Relevent logs
[<!-- Paste relevent logs if you have any -->](https://jenkins.jans.io/jenkins/job/jans-auth-server/13945/console)

### Possible root cause
Recent removal of clean up timer

--- a/jans-auth-server/server/src/test/java/io/jans/as/server/ws/rs/AuthorizationCodeFlowEmbeddedTest.java
+++ b/jans-auth-server/server/src/test/java/io/jans/as/server/ws/rs/AuthorizationCodeFlowEmbeddedTest.java
@@ -600,11 +600,9 @@ public class AuthorizationCodeFlowEmbeddedTest extends BaseTest {
 
         // Store current configuration
         int currentAuthorizationCodeLifetime = appConfiguration.getAuthorizationCodeLifetime();
-        int currentCleanServiceInterval = appConfiguration.getCleanServiceInterval();
 
         // We need to expire in test test code token faster than usual to avoid sleeping test for long time
         appConfiguration.setAuthorizationCodeLifetime(8);
-        appConfiguration.setCleanServiceInterval(6);
         try {
             final String state = UUID.randomUUID().toString();
 
@@ -656,7 +654,6 @@ public class AuthorizationCodeFlowEmbeddedTest extends BaseTest {
         } finally {
             // Restore configuration
             appConfiguration.setAuthorizationCodeLifetime(currentAuthorizationCodeLifetime);
-            appConfiguration.setCleanServiceInterval(currentCleanServiceInterval);
         }
     }
 

